css/style.css

```diff
@@ -0,0 +1,97 @@
+@media only screen and (max-width: 600px) {
+  body {
+    background-color: lightblue;
+    display: flex;
+  }
+}
+
+#navbar {
+  width: 100%;
+  background-color: brown;
+  position: fixed;
+  top: 0;
+}
+
+#navbar a {
+  float: right;
+  margin: 0 15px;
+  padding-bottom: 15px;
+  font-size: 1.8em;
+  color: rgb(255, 255, 255);
+  text-decoration: none;
+  text-align: center;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+}
+
+#navbar a:hover {
+  color: rgb(10, 10, 12);
+}
+
+#navbar a:visited {
+  text-decoration: none;
+}
+#welcome-section {
+  height: 100vh;
+  width: 100vw;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  color: rgba(250, 250, 250, 250);
+  background-color: rgba(50, 50, 50, 50);
+}
+
+h1 {
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+  font-size: 5em;
+}
+
+#projects {
+  width: 100%;
```

Comment:
![Screenshot from 2021-12-06 16-37-54](https://user-images.githubusercontent.com/62058680/144876117-6eb736e8-97e1-429f-a043-5f02132b1e98.png)
There is a hole in the project section
```suggestion
  I think it should be better to use grid or flexbox layout to fix this issue
```

Reply:
- [ ] Great job being polite in your reviews 👍 . But kindly follow my first comment, please. Let's use GitHub inline comments feature here too.